include/TGUI/Widgets/ChildWindow.hpp

```diff
@@ -399,8 +503,14 @@ namespace tgui
         Label          m_titleText;
         sf::Vector2f   m_draggingPosition;
         TitleAlignment m_titleAlignment = TitleAlignment::Center;
+        TitleButtons   m_titleButtons = TitleButtons::Close;
+        sf::String     m_closeButtonText = "x";
+        sf::String     m_minimizeButtonText = "-";
+        sf::String     m_maximizeButtonText = "+";
```

Comment:
They are also used inside the reload() function. It could be removed by changing a few other lines but I would just keep them. I'll have to rewrite the whole thing for 0.8 anyway so I'll see about what happens with these members then.